src/viam/app/data_client.py

```diff
@@ -437,6 +437,7 @@ async def binary_data_capture_upload(
         method_parameters: Optional[Mapping[str, Any]] = None,
         tags: Optional[List[str]] = None,
         data_request_times: Optional[Tuple[datetime, datetime]] = None,
+        file_extension: Optional[str] = None,
```

Comment:
Can/does the file extension get translated to mime_type in the cloud?

Reply:
According to @agreenb, [yes](https://viam.atlassian.net/browse/DATA-2034?focusedCommentId=22732)

> ... which we convert to mime type in the backend

Reply:
Yep! [here](https://github.com/viamrobotics/app/blob/aa701573f796faf5398b51b229c4edf68aa4a023/datasync/data_capture_upload.go#L238) and [here](https://github.com/viamrobotics/app/blob/aa701573f796faf5398b51b229c4edf68aa4a023/datasync/server.go#L135-L165), which uses a [.pcd](https://github.com/viamrobotics/app/blob/aa701573f796faf5398b51b229c4edf68aa4a023/datasync/server.go#L35) file extension to get the point cloud mime type.